manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -541,7 +541,10 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             ,name: 'template'
             ,id: 'modx-resource-template'
             ,anchor: '100%'
-            ,editable: false
+            ,editable: true
+            ,typeAhead: true
```

Comment:
I suggest disabling `typeAhead` function which preselects the first item. When you start typing `template` and wait for suggestions, it automatically fills `BaseTemplate` and you can't continue writing